libnavui-maneuver/api/current.txt

```diff
@@ -406,10 +404,10 @@ package com.mapbox.navigation.ui.maneuver.view {
   }
 
   public final class MapboxUpcomingManeuverAdapter.MapboxUpcomingManeuverViewHolder extends androidx.recyclerview.widget.RecyclerView.ViewHolder {
-    ctor public MapboxUpcomingManeuverAdapter.MapboxUpcomingManeuverViewHolder(android.view.View view);
+    ctor public MapboxUpcomingManeuverAdapter.MapboxUpcomingManeuverViewHolder(error.NonExistentClass viewBinding);
```

Comment:
Could you cut a follow-up ticket to investigate how to add these `viewBinding` auto-generated classes to the classpath so Metalava sees them and avoid `error.NonExistentClass`?